### PR TITLE
Friends leaderboard

### PR DESCRIPTION
**Pull request by alberttkt**

### CODE REVIEW

File: app/src/main/java/ch/epfl/culturequest/social/Profile.java [hunk 1]
```diff
@@ -50,17 +53,19 @@ public Profile(String username, String profilePicture) {
         this.profilePicture = profilePicture;
         this.images = List.of();
         this.score = 0;
+        this.Friends = new ArrayList<>();
     }
 
-    public Profile(String uid, String name, String username, String email, String phoneNumber, String profilePicture, List<Image> images, Integer score) {
+    public Profile(String uid, String name, String username, String email, String phoneNumber, String profilePicture, ArrayList<String> friends, Integer score) {
```
Comment (lucamouchel) [inline app/src/main/java/ch/epfl/culturequest/social/Profile.java:59]: why did you remove the images attribute in the constructor?

**Review by lucamouchel — COMMENTED** [review]
everything sgtm, just one remark on profile constructor but all good otherwise

File: app/src/main/java/ch/epfl/culturequest/social/Profile.java (hunk 1 above)
Comment (alberttkt) [reply app/src/main/java/ch/epfl/culturequest/social/Profile.java:59]: Because the Profile won't have images anymore since it has been replaced by posts

**Review by lucamouchel — APPROVED** [review]
